docs/source/deep_dives/checkpointer.rst

```diff
@@ -274,6 +266,69 @@ for testing or for loading quantized models for generation.
 
 |
 
+Checkpoint Output
+---------------------------------
+
+Congrats for getting this far! Let's say you have followed our :ref:`End-to-End Workflow with torchtune <e2e_flow>` and trained a llama 3.2 3B using one of our LoRA recipes.
+
+Now let's visualize the outputs. A simple way of doing this is by running `tree -a path/to/outputdir`, which should show something like the tree below.
+There are 3 types of folders:
+
+1) **recipe_state**: Holds recipe_state.pt with the information necessary to restart the last intermediate epoch. More on that later;
+2) **logs**: Defined in your config in metric_logger;
+3) **epoch_{}**: Contains your new trained model weights plus all original files of the model, except the checkpoints, making it easy for you to choose an specific epoch to run inference on or push to a model hub;
```

Comment:
So this is duplicated in the E2E tutorial? Why not just link? If not, would adopt the same comments here

Reply:
I think it’s fine to duplicate this part, but each tutorial mentions the other